src/checkpoint/dispatch/dispatch.impl.h

```diff
@@ -106,6 +105,36 @@ TraverserT& withTypeIdx(TraverserT& t) {
   return t;
 }
 
+template <typename T, typename TraverserT>
+TraverserT& withMemUsed(TraverserT& t, SerialSizeType len) {
+  using CleanT = typename CleanType<SerialSizeType>::CleanT;
```

Comment:
I don't like having another `CleanT` below that shadows this. I think it might be confusing.

I'm also confused why this statement is cleaning the `SerialSizeType`, which is a very simple type that basically is `size_t` or something. What am I missing here?

Reply:
No, that's completely redundant, you right! Removed.